src/modules/powerrename/testapp/PowerRenameTest.vcxproj

```diff
@@ -185,6 +185,11 @@
   <ItemGroup>
     <ResourceCompile Include="PowerRenameTest.rc" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
If it's possible to not reference the `common` project and just add an Include path to it, let's do that!

Reply:
I am not sure if that can be done as it is a dynamic library(.dll). Also, this change was made in the last PR which has been merged to master and my branch was behind (#1101)